readme.md

```diff
@@ -47,6 +47,12 @@ Yields:
 { type: 'leaf', value: '3' }
 ```
 
+**For beginners** Don't forget to install all dependencies for this example: 
```

Comment:
This isn't just for beginners, but can benefit anyone running the example, maybe phrasing something like:

```suggestion
Note: this example also uses `unist-builder`, to run the example ensure both `unist-builder` and `unist-util-visit` are installed:
```